_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -643,17 +681,17 @@ Sometimes plots like this are called "spaghetti plots" because all the lines loo
 ## Discrete Plots
 _[Back to top](#contents)_
 
-So far we've looked at two plot types (`geom_point` and `geom_line`) which work when both the x and y values are numeric. But sometimes one of your values may be discrete (a factor or character).
+So far we've looked at two plot types (`geom_point` and `geom_line`) which work when both the x and y values are continuous. But sometimes one of your values may be discrete. 
 
-We've previously used the discrete values of the `continent` column to color in our points and lines. But now let's try moving that variable to the `x` axis. Let's say we are curious about comparing the distribution of the life expectancy values for each of the different continents for the `gapminder_1997` data. We can do so using a box plot. Try this out yourself in the exercise below!
+We've previously used the discrete values of the `continent` column to color in our points and lines. But now let's try moving that variable to the `x` axis. Let's say we are curious about comparing the distribution of the lung cancer rates for each of the different continents for the `smoking` data. We can do so using a box plot. Try this out yourself in the exercise below!
 
 > ## Box plots
-> Using the `gapminder_1997` data, use ggplot to create a box plot with continent on the x axis and life expectancy on the y axis. You can use the examples from earlier in the lesson as a template to remember how to pass ggplot data and map aesthetics and geometries onto the plot. If you're really stuck, feel free to use the internet as well!
+> Using the `smoking` data, use ggplot to create a box plot with continent on the x axis and lung cancer rates on the y axis. You can use the examples from earlier in the lesson as a template to remember how to pass ggplot data and map aesthetics and geometries onto the plot. If you're really stuck, feel free to use the internet as well!
```

Comment:
Maybe we should use the `smoking_1990` dataset here? That will give us fewer points and it kind of feels more interpretable?